### PR TITLE
[4.0] Core.js cleanup Pt1

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- The function `Joomla.popupWindow()` moved to toolbar.js where it belongs
- The function `window.writeDynaList()` moved to system\fields\moduleorder.js where it belongs
- The function `window.changeDynaList()` was removed completely as it had no real usage
- The function `window.radioGetCheckedValue()` was removed completely as it had no real usage

- All the code in core.js was just muted (to help the syncing with 3.x)

### Testing Instructions
Go to Administrator-> articles and press the help button on the toolbar, you should see a new window (with no toolbar and an error message, if so then this PR works)

### Documentation Changes Required
Deprecations need to be done in the 3.X branch

@mbabker @wilsonge any objections here?

### CODE REVIEW

File: media/system/js/core.js [hunk 1]
```diff
@@ -528,19 +528,19 @@ Joomla.editors.instances = Joomla.editors.instances || {
...
-			    ',width=' + w +
-			    ',top=' + wint +
-			    ',left=' + winl +
-			    ',scrollbars=' + scroll +
-			    ',resizable';
-
-		window.open( mypage, myname, winprops )
-			.window.focus();
-	};
+	// Joomla.popupWindow = function( mypage, myname, w, h, scroll ) {
```
Comment (zero-24) [inline media/system/js/core.js]: do you want to keep the commented code?

Comment (dgrammatiko) [reply media/system/js/core.js]: I don't know if that will help George to keep the file synced. I can remove them completely

**Comment by zero-24** [comment]
> The function window.writeDynaList() moved to system\fields\moduleorder.js where it belongs

hmm looks like this file is not changed in this PR?

**Comment by dgrammatiko** [comment]
@zero-24 check that file, the function already exists there, I just hadn't cleared it from core.js

**Comment by zero-24** [comment]
hmm that is a completly different code + signature?
https://github.com/joomla/joomla-cms/blob/4.0-dev/media/system/js/fields/moduleorder.js#L30

**Comment by dgrammatiko** [comment]
@zero-24 it is, because it's a local function, not a global like it was before ;)

Also check the commit: https://github.com/joomla/joomla-cms/commit/3d976e4e47362229e9ade6ccd4e01c44be54dec6

File: media/system/js/toolbar.js [hunk 2]
```diff
@@ -3,16 +3,43 @@
...
+		window.open( mypage, myname, winprops ).window.focus();
+	};
+
 	document.addEventListener('DOMContentLoaded', function() {
+		/**
+		 * Fix the alignment of the Options and Help toolbar buttons
+		 */
 		var toolbarOptions = document.getElementById('toolbar-options'),
-		    toolbarHelp    = document.getElementById('toolbar-help');
+			toolbarHelp    = document.getElementById('toolbar-help');
```
Comment (C-Lodder) [inline media/system/js/toolbar.js]: indentation.

File: media/system/js/core.js (hunk 1 above)
Comment (C-Lodder) [reply media/system/js/core.js]: May as well remove them completely

Comment (dgrammatiko) [reply media/system/js/core.js]: Well the first time I did that and it didn't work out very well: https://github.com/joomla/joomla-cms/commit/3d976e4e47362229e9ade6ccd4e01c44be54dec6

Comment (wilsonge) [reply media/system/js/core.js]: As soon as you make a change like this it causes conflicts. Removing it or commenting out doesn't make a huge difference to be honest.

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 86076482edf10649ab1b8005774fa560fdef7165<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17362">issues.joomla.org/tracker/joomla-cms/17362</a>.</sub>

**Pull request merged**